README.md

```diff
@@ -45,6 +45,8 @@ This file contains a number of front-end interview questions that can be used wh
 * Explain what ARIA and screenreaders are, and how to make a website accessible.
 * Explain some of the pros and cons for CSS animations versus JavaScript animations.
 * What does CORS stand for and what issue does it address?
+* What is the difference between merging vs. rebasing?
+* How do you stay up to date on technology and the what is happening in the technology industry?
```

Comment:
This feels like an alternative to, or a follow up of `What did you learn yesterday/this week?`

If this were to be added, I'd suggest putting it directly under the previously stated question.